venues/DBLP.org/python/superuser-init.py

```diff
@@ -112,7 +112,7 @@ def overwrite_allowed(groupid):
                 'description': 'Comma separated list of author email addresses, in the same order as above.',
                 'order': 3,
                 'values-regex': "[^;,\\n]+(,[^,\\n]+)*",
-                'required':True
+                'required':False
```

Comment:
I'm not sure. It's technically not a "submission". It might be a good idea to name it "DBLP.org/-/Upload".

It's not clear to me, since we haven't fully explored this use case for the system, but that would be what I would call it...